.github/workflows/full-integration-tests.yml

```diff
@@ -92,7 +92,7 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       with:
         status: ${{ job.status }}
-        channel: "#terra-wsm-alerts"
+        channel: "#dsp-workspace-test-alerts"
```

Comment:
I think this is #dsp-workspaces-test-alerts (note the plural)

Reply:
Ah, good catch! I copied this from a BPM github action, so I'll update it there also.